### PR TITLE
LPS-79417 Using default version of portal kernel

### PR DESCRIPTION
**Pull request by 4lejandrito**
Hi @gamerson,

Thanks for merging my template. 

I have tested it and it almost worked. The problem is that `portal kernel` is now `3.0.0` and the module generated by the archetype specifies `2.66.0`. I have changed it to default (https://github.com/gamerson/liferay-portal/pull/125/commits/85f6032f2c7b3c8499627536b77e7358f4690161) but I am not sure if this is correct for a template.

Thanks!

### CODE REVIEW

File: modules/sdk/project-templates/project-templates-social-bookmark/src/main/resources/archetype-resources/build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@ apply plugin: "com.liferay.plugin"
 #end
 dependencies {
 	compileOnly group: "com.liferay", name: "com.liferay.social.bookmarks.api", version: "1.0.0-SNAPSHOT"
-	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "2.54.0"
+	compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "default"
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-social-bookmark/src/main/resources/archetype-resources/build.gradle:18]: hey @4lejandrito  we can't use `default` for the version.  That only works for internal build.  Not external or client (workspace) projects.

File: modules/sdk/project-templates/project-templates-social-bookmark/src/main/java/com/liferay/project/templates/social/bookmark/internal/SocialBookmarkProjectTemplateCustomizer.java [hunk 2]
```diff
@@ -1,54 +0,0 @@
-/**
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-social-bookmark/src/main/java/com/liferay/project/templates/social/bookmark/internal/SocialBookmarkProjectTemplateCustomizer.java:1]: Hey quick question, why did you remove this class?  From my understanding it is needed.

Comment (4lejandrito) [reply modules/sdk/project-templates/project-templates-social-bookmark/src/main/java/com/liferay/project/templates/social/bookmark/internal/SocialBookmarkProjectTemplateCustomizer.java:1]: Hi @gamerson 

We are no longer using that parameter. Everything is derived from the `className`. I did remove it in https://github.com/christopherbryanboyd/liferay-portal/pull/11/commits/bbd5d11fc14f944f7921420bcb60da6cea9125a0 but I reckon you guys squashed it or did it yourselves. The fact is that there is no trace of the `socialBookmarkType` apart from that class therefore it is not needed.

File: modules/sdk/project-templates/project-templates-social-bookmark/src/main/resources/archetype-resources/build.gradle (hunk 1 above)
Comment (4lejandrito) [reply modules/sdk/project-templates/project-templates-social-bookmark/src/main/resources/archetype-resources/build.gradle:18]: Can we at least add a version range? Something like:

```groovy
compileOnly group: "com.liferay.portal", name: "com.liferay.portal.kernel", version: "[2.0.0,3.0.0]"
```

Comment (gamerson) [reply modules/sdk/project-templates/project-templates-social-bookmark/src/main/resources/archetype-resources/build.gradle:18]: No that doesn't work for gradle.  So you would need to use the lowest possible portal kernel version that you can.  So 3.0.0 or 2.x.x, whatever is lowest.

File: modules/sdk/project-templates/project-templates-social-bookmark/src/main/java/com/liferay/project/templates/social/bookmark/internal/SocialBookmarkProjectTemplateCustomizer.java (hunk 2 above)
Comment (gamerson) [reply modules/sdk/project-templates/project-templates-social-bookmark/src/main/java/com/liferay/project/templates/social/bookmark/internal/SocialBookmarkProjectTemplateCustomizer.java:1]: Oh i see, Yeah we can remove it.

File: modules/sdk/project-templates/project-templates-social-bookmark/src/main/resources/archetype-resources/build.gradle (hunk 1 above)
Comment (gamerson) [reply modules/sdk/project-templates/project-templates-social-bookmark/src/main/resources/archetype-resources/build.gradle:18]: Since this template is only available for 7.1, we can move this to 3.0.0 I think.

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/Ithildir/liferay-portal/pull/1017. [See changes here.](https://github.com/gamerson/liferay-portal/compare/dfbf8e11570e~1...dfbf8e11570e) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
